### ISSUE TITLE
Twig global variables are not injected via configuration

### ISSUE THREAD

**Issue by ragusa87** [GeneralMediaCH/TwigEmailRenderBundle#4]
Twig's configuration "global variable" and "paths" are injected into the twig service via `Symfony\Bundle\TwigBundle\DependencyInjection\TwigExtension.php`.
Unforunately, injection is not made for the twigEmailRenderService.

So globale variable and search path are undefined into the twig extension.

TODO: Fix by:
- Clone the main Twig class
- Add a compiler pass to inject the same values by parsing configuration (or container definition?)

If you use something like this, it will not work.
```yml
twig:
  globals:
    global_locale: '%locale%'
```

**Comment by ragusa87** [comment]
Try this: 
```php
namespace Generalmedia\AdminAccrowinBundle\DependencyInjection\Compiler;


use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
use Symfony\Component\DependencyInjection\ContainerBuilder;

class TwigEmailRendererInjectionCompiler implements CompilerPassInterface
{
    const EMAIL_RENDER = "generalmedia_email_render_twig";

    /**
     * You can modify the container here before it is dumped to PHP code.
     *
     * @param ContainerBuilder $container
     */
    public function process(ContainerBuilder $container)
    {
        foreach (["twig", self::EMAIL_RENDER] as $requiredService) {
            if (false === $container->hasDefinition($requiredService)) {
                return;
            }
        }

        $renderDef = $container->getDefinition(self::EMAIL_RENDER);
        foreach ($container->getDefinition("twig")->getMethodCalls() as $methodCall) {
            if ("addGlobal" !== $methodCall[0]) {
                continue;
            }
            $renderDef = $renderDef->addMethodCall($methodCall[0], $methodCall[1]);
        }
        $container->setDefinition(self::EMAIL_RENDER, $renderDef);
    }
}
```

**Comment by ragusa87** [comment]
Solved in 2.0.0
